packages/vaadin-upload/src/vaadin-upload.js

```diff
@@ -76,6 +76,11 @@ class UploadElement extends ElementMixin(ThemableMixin(PolymerElement)) {
         [hidden] {
           display: none !important;
         }
+
+        ul[part='file-list'] {
```

Comment:
I would suggest to not add element tag here (part is enough). Otherwise users creating custom themes might need to increase specificity, in case if they need to adjust custom padding or margin on the file list.